sdcm/nemesis.py

```diff
@@ -50,7 +50,7 @@
 from sdcm.utils.common import (get_db_tables, generate_random_string,
                                update_certificates, reach_enospc_on_node, clean_enospc_on_node,
                                parse_nodetool_listsnapshots,
-                               update_authenticator, ParallelObject)
+                               update_authenticator, ParallelObject, get_table_clustering_order)
```

Comment:
where is this function coming from?
from another PR?

it is failing the precommit hooks

Reply:
yes, see explanation above: https://github.com/scylladb/scylla-cluster-tests/pull/4274#issue-810523521